src/coreclr/tests/src/dirs.proj

```diff
@@ -45,6 +45,10 @@
       </_GroupStartsWith>
     </ItemGroup>
 
+    <PropertyGroup>
+      <__TestGroupToBuild Condition="'$(__TestGroupToBuild)' == ''">1</__TestGroupToBuild>
```

Comment:
Where was this defined before? Or is it to avoid having to pass it down?

Reply:
right, otherwise msbuild would complain. Currently this is always invoked from scripts but we will change this in the future. So this is just to unblock invoking this project directly in the meantime.